### PR TITLE
MAGN-6096 Revit nodes, when unavailable, show as "Legacy Node", not as "Dummy" node.

### PR DESCRIPTION
**Pull request by ikeough**
This PR fixes a regression with DummyNodes, detailed in [MAGN-6096](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6096). 

- The resource path string for the unresolved node image was incorrect due to the renaming of CoreNodeModelsWpf. When the resource could not be found, the dummy node image was not applied to the nodes. The resource path has been updated to reference the renamed assembly.
- NodeModel types that could not be resolved through the migration manager created the proper Dummy node, but ZT types that could not be resolved by the ZeroTouchNodeLoader, created a DummyNode object, but did not set its location. Dummy nodes were placed at the origin and were not immediately visible to the user as most graphs are zoomed to their original content. The X and Y coordinates are now read from the node's original xml element and applied to properly position the dummy node.
- Dummy nodes created during a failure to resolve a ZT type were incorrectly nicknamed "Legacy Node". The nickname of the node is now set in the DummyNode constructor.

image

PTAL:
- [x] @pboyer

### CODE REVIEW

File: src/DynamoCore/Models/NodeLoaders/ZeroTouchNodeLoader.cs [hunk 1]
```diff
@@ -70,13 +71,20 @@ public NodeModel CreateNodeFromXml(XmlElement nodeElement, SaveContext context)
...
+
+                var dummy = new DummyNode(
                     inputcount,
                     1,
                     nickname,
                     nodeElement,
                     assembly,
                     DummyNode.Nature.Unresolved);
+
+                var helper = new XmlElementHelper(nodeElement);
```
Comment (sharadkjaiswal) [inline src/DynamoCore/Models/NodeLoaders/ZeroTouchNodeLoader.cs]: #3663 is also trying to do this inside DummyNode constructor.

File: src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DummyNode.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public void CustomizeView(DummyNode model, Dynamo.Controls.NodeView nodeView)
             if (model.NodeNature == DummyNode.Nature.Unresolved)
                 fileName = "MissingNode.png";
 
-            var src = @"/DSCoreNodesWpf;component/Resources/" + fileName;
+            var src = @"/CoreNodeModelsWpf;component/Resources/" + fileName;
```
Comment (pboyer) [inline src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DummyNode.cs]: Yep.  I hope I didn't miss any others!

File: src/DynamoCore/Nodes/DummyNode.cs [hunk 3]
```diff
@@ -44,6 +42,10 @@ public DummyNode(int inputCount, int outputCount, string legacyName, XmlElement
             ShouldDisplayPreviewCore = false;
 
             UpdatePorts();
+
+            var helper = new XmlElementHelper(originalElement);
+            X = helper.ReadDouble("x", 0.0);
+            Y = helper.ReadDouble("y", 0.0);
         }
```
Comment (pboyer) [inline src/DynamoCore/Nodes/DummyNode.cs]: Nice.

**Comment by pboyer** [comment]
@ikeough, This appears fine to me. :)

**Pull request merged**